### ISSUE TITLE
Update NMDC Schema to accommodate Study Landing Page updates

### ISSUE THREAD

**Issue by brynnz22** [microbiomedata/issues#342]
The following potential nmdc schema updates have been identified to accommodate the study landing page updates:

- [ ] https://github.com/microbiomedata/issues/issues/338
- [ ] https://github.com/microbiomedata/issues/issues/339
- [ ] https://github.com/microbiomedata/issues/issues/340
- [ ] https://github.com/microbiomedata/issues/issues/341

**Comment by ssarrafan** [comment]
@brynnz22 I'm assuming these will not be completed by tomorrow.  Also assuming you're actively working on these for the next sprint?  Can you please confirm this? 

Nice checkboxes!

**Comment by brynnz22** [comment]
@ssarrafan These will not be completed by tomorrow. The first one, the doi updates is being reviewed right now, see the [PR](https://github.com/microbiomedata/nmdc-schema/pull/1039). The others, I am still working on!

**Comment by ssarrafan** [comment]
@brynnz22 assuming this is still an active issue that you're working on so will move to the next sprint. Let me know if should go in the backlog instead.

**Comment by ssarrafan** [comment]
Issue doesn't appear to be active. Removing from sprint and adding backlog label.  @brynnz22 @mslarae13

**Comment by brynnz22** [comment]
Hi @ssarrafan I'm very sorry for missing your comments! This is actively being worked on. A lot of the sub-issues are in review right now, so I will move this back to In progress. The first three sub-issues are in review. I still need to fix the publications (the fourth issue), and I suspect the fifth issue about collapsing the protocols will not be necessary.

**Comment by brynnz22** [comment]
@ssarrafan I'm very sorry for missing your comments. The first three sub-issues are in review. They just need to be merged into main. The fourth issue I still need to work on, and I suspect the last issue will get canceled. I will move things accordingly.

**Comment by brynnz22** [comment]
Waiting on the doi updates to get merged before I can work on the publication updates.

**Comment by ssarrafan** [comment]
@brynnz22 @naglepuff Mike mentioned he was almost done with the DOI updates yesterday so will assume this needs to continue to the next sprint for the publication updates.

**Comment by ssarrafan** [comment]
DOI still in the works so will move again to the next sprint. Let me know if it should go in the backlog instead @brynnz22 .

**Comment by ssarrafan** [comment]
@brynnz22 will move this one more time but would be great to get an update on if it's active or not.  I'll assume it's going in the backlog if it's still open at the end of this sprint.

**Comment by ssarrafan** [comment]
I think this is in progress. Will move this again. @brynnz22 if you can add an update I'd appreciate it.

**Comment by brynnz22** [comment]
Thank you. Currently working on adding the new `Doi` class and the accompanying migration code. Also waiting for dev testing the umbrella studies to re-add the umbrella studies.

**Comment by turbomam** [comment]
Hi, @brynnz22 . Can you please point me to a branch where umbrella studies are being modeled?

**Comment by brynnz22** [comment]
@turbomam umbrella studies have already been modeled. You can look in the main branch. We use the `part_of` slot in the `Study` class to model this.

**Comment by turbomam** [comment]
Thanks @brynnz22 

The text "umbrella project" only appears once in the merged schema, as an `alises` for `Study`. Is that reasonable? Is "umbrella project" a good alias for all Studies, or just a subset?

Is the differentiating characteristic that umbrella projects have at lest one Study part? If so, plese add that as a comment on `Study`.

**Comment by brynnz22** [comment]
Waiting on nmdc-runtime to fix https://github.com/microbiomedata/nmdc-runtime/issues/340 so I can re-add the umbrella studies. Will move this to the next spring.

**Comment by brynnz22** [comment]
Waiting for the runtime fix to add the umbrella studies through the json submit endpoint. Will work on the DOI pattern for the `websites` slot and adding the data management plan as an enum to `doi_category` next sprint.

**Comment by brynnz22** [comment]
Closing this issue since all of the sub-issues for Study schema updates are now complete.
